PrimeChecker.js

```diff
@@ -0,0 +1,12 @@
+const primeChecker = (number) => {
+  if (!number) return "Please input a number";
+  if (number <= 1) return false;
+  let isPrime = true;
+  for (let i = 2; i < number; i++) {
+    if (number % i == 0) {
+      isPrime = false;
+      break;
+    }
+  }
+  return isPrime;
+};
```

Comment:
Good implementation, but you can return false immediately after finding that a number is divisible by 2, instead of assigning a boolean to isPrime and return it later. 
I can see you immediately break out of the loop but, directly returning would be better